### PR TITLE
Minor docstring fixes for #4150

### PR DESCRIPTION
**Pull request by jni**
## Description

Additionally, I made mask keyword-only.

## Checklist

- [ ] [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- [ ] Gallery example in `./doc/examples` (new features only)
- [ ] Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- [ ] Unit tests
- [ ] Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)

## For reviewers

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.
- [ ] Consider backporting the PR with `@meeseeksdev backport to v0.14.x`

### CODE REVIEW

File: skimage/measure/_find_contours_cy.pyx [hunk 1]
```diff
@@ -25,7 +25,7 @@ def iterate_and_store(double[:, :] array,
     nonzero, high-values pixels are considered to be face+vertex
     connected into objects; otherwise low-valued pixels are.
 
-    Positions where the boolean array mask is False are considered
+    Positions where the boolean array ``mask`` is ``False`` are considered
```
Comment (lagru) [inline skimage/measure/_find_contours_cy.pyx]: Do you deliberately choose to format variables as literals in docstrings? The [convention (section Parameters)](https://numpydoc.readthedocs.io/en/latest/format.html#sections) I know is to emphasize with a single surrounding \`, e.g. \`mask\`.

Comment (lagru) [reply skimage/measure/_find_contours_cy.pyx]: Not to block this PR. I'm just curious.

Comment (jni) [reply skimage/measure/_find_contours_cy.pyx]: You're correct! I've gotten into the habit of double backticks as they are always safe. Fix coming...

Comment (jni) [reply skimage/measure/_find_contours_cy.pyx]: Fixed in cde8c3137

**Comment by sciunto** [comment]
But we say that there is not rendering, and I find this problematic... #4181 4117

**Comment by jni** [comment]
Oops! A bug in the numpydoc docs???

**Comment by sciunto** [comment]
@jni It's not what you said here: https://github.com/scikit-image/scikit-image/issues/4117#issuecomment-524660492 :)

**Comment by jni** [comment]
Yes, that's the Sphinx docs. @lagru points to the NumPy docstring conventions page and they explicitly say, use single-ticks.

**Comment by sciunto** [comment]
I don't know whether it is deliberate on the numpy doc side.

**Comment by jni** [comment]
> I don't know whether it is deliberate on the numpy doc side.

That was my implication when I said "bug" combined with a triple question mark. =P

**Comment by sciunto** [comment]
I open an issue on the numpydoc repository, they can probably shed some light on the mystery. :)

**Comment by sciunto** [comment]
See https://github.com/numpy/numpydoc/issues/233

**Comment by lagru** [comment]
> See numpy/numpydoc#233 

The way I see it this is an issue with the configuration and/or the theme (?) of the skimage docs. Correct? E.g. [SciPy's docs render it correctly](https://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.argrelextrema.html).

As long as we try to be consistent I'm fine with either solution. Double backticks is probably the better option as long as the docs don't render it correctly.

**Comment by lagru** [comment]
Thanks @jni!

**Comment by lagru** [comment]
> Sure. Until that's fixed, double backticks make more sense. PRs welcome!

There's already #4188.

**Comment by jni** [comment]
... Where you suggested it's not the right solution???

**Comment by lagru** [comment]
Oh, perhaps I worded that badly. I'll try address that.

**Comment by stefanv** [comment]
Single backticks for parameter names, double for code quotes.

**Pull request merged**